server/servercontext.py

```diff
@@ -17,6 +17,9 @@
 from .protocol import DisconnectedError, Protocol, QDataStreamProtocol
 from .types import Address
 
+MiB = 2 ** 20
+LIMIT = 10 * MiB
```

Comment:
Any point in making this a config value? We couldn't change it without restarting the server anyway.

Reply:
I don't think so really. Only if this has some non minimal impact but I doubt it

Reply:
creating config values that need a restart anyway seems counterintuitive to me. Additionally I don't think there is much value in being able to easily change this value